meerkat_libs/db_adapters/postgresql.py

```diff
@@ -0,0 +1,246 @@
+import psycopg2
+import logging
+import json
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+from psycopg2 import sql
+from psycopg2.extras import Json
+
+
+class PostgreSQLAdapter():
+
+    def __init__(self, db_name, structure, user='postgres', host='db'):
+        self.db_name = db_name
+        self.structure = structure
+        self.user = user
+        self.host = host
+        self.keys = self._extract_keys()
+
+    def drop(self):
```

Comment:
Rename to `drop_all_tables`?

Reply:
I wanted this name to follow similar naming convention as the other functions, which don't give this level of detail: e.g. "setup". This function is to drop what ever needs to be dropped to start over again in the particular database we are working with. The process for doing this might differ from DB to DB. I suppose that the word "drop" is an SQL command and that might be confusing and makes people ask the question "drop what exactly?".  I suggest renaming to "clear".

Reply:
Ah maybe you're right.  This is actually simple enough to change.  Will make the change.